composer.json

```diff
@@ -31,7 +31,9 @@
         "dmk/mklib": ">=9.5.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2@stable"
+        "friendsofphp/php-cs-fixer": "~2@stable",
+        "nimut/testing-framework": "^1.0 || ^4.0",
+        "phpunit/phpunit": "^4.7 || ^5.0 || ^7.0"
```

Comment:
This constraints is useless. The `nimut/testing-framework` requires at least `phpunit >=6.0`. So `4.7` and `5.0` could be dropped.